.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Welcome to the Julia project!
+
+We use the GitHub issue tracker for bug reports and feature requests.
+If you have a question or are unsure if the behavior you're experiencing is a bug,
+please post to [our Discourse site](https://discourse.julialang.org).
```

Comment:
I think @nalimilan raises good points and I'd agree with his proposal (pointing to the Community page instead of singling out Discourse).